### DISCUSSION TITLE
Launch plan

### DISCUSSION THREAD

**Discussion by kmalhi** [perses/perses#1297] [General]
Perses is being launched soon at PromCon. We need a clear launch plan with some goals in mind. What do we want out of the launch? Examples could be , we should be able to attract 2 new contributors to Perses, or get 50 unique users to demo.perses or 5 new users download Perses and give feedback. 
Based on the goal, we can decide and prioritize what things we need to work on. Thoughts?

**Comment by sjcobb** [comment]
All 3 of your examples make sense to me and seem realistic! The last one about getting people to actually use the app and give early feedback seems the most important though at this stage so figuring out the plan for tasks like what @nicolastakashi posted to build a Helm chart #1270 will be great. 

We should probably have a more formal “bug bash” too once all the project / global variables and datasource CRUD PRs land so that we fix all the most important issues before PromCon

**Comment by Nexucis** [comment]
On my side launching Perses means:
* We give a message that the project reachs a point where we are sure about the architecture, the different technologies and it can be tested
* Usage in production is unlikely until we get a certain amount of feedback to know if we will have to break things and until we find a good strategy for the plugins evolution / upgrade

I think we should be humble in our expectations about the launch and don't expect too much things.
* Having feedback would be already a great things
* Having contributors would be amazing

We need to be sure everything is on place to be able to try Perses with easy install, easy usage.
For example, does the helm chart can be used ? Do we miss something to run the command `helm install perses` ?

Critical bugs in the usage of the application should be the priority I think. But from what I saw and understood so far, we don't have any so that's a really good things !
Moreover the team did an incredible job, and we have (almost or already) every required feature to use the application and now it's more about making the experience great :)

**Reply by Nexucis** [reply]
I proposed a topic at the PromCon. Let's see if it is accepted.

**Comment by nicolastakashi** [comment]
For the helm chart, I'm trying to contact the user is owning the a repo on artifact hub named perses, but until now I don't have any feedback from him.

Maybe worth to talk to artifact hub, wdyt?

**Reply by celian-garcia** [reply]
Yep indeed, @Nexucis do you think you can help on that? The perses name is owned by someone in artifact hub who doesn't maintain it. I'm assuming that a proof that Perses is something official and used would make them remove the unused repository WDYT ?

**Reply by Nexucis** [reply]
Let's try that then :)
